### PR TITLE
[EXAMPLE] Add example to load another BPMN diagram in the same container on double click on a call activity

### PR DESCRIPTION
**Pull request by csouchet**
https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/call_activity_reload_dblclick/examples/index.html

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ Go to the repository root and use one the following solutions
 
 ### basic examples
 
-- [getting started](examples/01-getting-started/README.md) - the simplest way to integrate `bpmn-visualization` in an HTML page
-- [load local BPMN diagrams](examples/load-local-bpmn-diagrams/README.md) - load BPMN diagrams stored on your local device
-- [load remote BPMN diagrams](examples/load-remote-bpmn-diagrams/README.md) - load BPMN diagrams from the [bpmn-miwg-test-suite GitHub repository](https://github.com/bpmn-miwg/bpmn-miwg-test-suite)
+- [Getting started](examples/01-getting-started/README.md) - the simplest way to integrate `bpmn-visualization` in an HTML page
+- [Load local BPMN diagrams](examples/load-local-bpmn-diagrams/README.md) - load BPMN diagrams stored on your local device
+- [Load remote BPMN diagrams](examples/load-remote-bpmn-diagrams/README.md) - load BPMN diagrams from the [bpmn-miwg-test-suite GitHub repository](https://github.com/bpmn-miwg/bpmn-miwg-test-suite)
```
Comment (tbouffard) [inline README.md:64]: :warning: not related to this PR, probably more in #109

File: README.md [hunk 2]
```diff
@@ -73,24 +73,25 @@ the viewport horizontally or vertically
 They are mainly hacks to let you see what will be later available in a more integrated way. Extensibility features will
 be progressively added to `bpmn-visualization`. See the [`Extensions` Milestone](https://github.com/process-analytics/bpmn-visualization-js/milestone/13).
 
-- [custom user task icon](./examples/custom-user-task-icon/README.md) - use your own icon
-- [custom colors](examples/custom-colors/README.md) - custom defaults, specific to BPMN element types
-- [custom fonts](examples/custom-fonts/README.md) - custom defaults, specific to BPMN element types
+- [Custom user task icon](./examples/custom-user-task-icon/README.md) - use your own icon
+- [Custom colors](examples/custom-colors/README.md) - custom defaults, specific to BPMN element types
+- [Custom fonts](examples/custom-fonts/README.md) - custom defaults, specific to BPMN element types
```
Comment (tbouffard) [inline README.md:78]: :warning: not related to this PR, probably more in #109

File: README.md [hunk 3]
```diff
@@ -73,24 +73,25 @@ the viewport horizontally or vertically
...
 #### interaction examples
 
 - [Open a call activity from a main BPMN Diagram on a modal](examples/interaction/call-activity-with-modal-on-mouse-over/README.md) - interact with a diagram to open the details of a Call Activity on a modal
 - [Open a call activity from a main BPMN Diagram on a tab (of the page)](examples/interaction/call-activity-with-tabs-on-click/README.md) - interact with a diagram to open the details of a Call Activity in a tab (of the page)
+- [Open a call activity from a main BPMN Diagram on the same container](./examples/interaction/call-activity-with-reload-on-dblclick/README.md) - interacted with a diagram to load the details of a Call Activity on the same container
 
 #### miscellaneous examples
 
-- [compare `bpmn-visualization` with `bpmn-js`](./examples/misc/compare-with-bpmn-js/README.md) - compare the libraries on BPMN elements rendering, navigation and API usage
+- [Compare `bpmn-visualization` with `bpmn-js`](./examples/misc/compare-with-bpmn-js/README.md) - compare the libraries on BPMN elements rendering, navigation and API usage
```
Comment (tbouffard) [inline README.md:90]: :warning: not related to this PR, probably more in #109

File: README.md [hunk 4]
```diff
@@ -73,24 +73,25 @@ the viewport horizontally or vertically
...
 
 ### `bpmn-visualization` usage in projects
 
-- [typescript + rollup](examples/projects/typescript-vanilla-with-rollup/README.md) - integration in a Vanilla Typescript rollup project
+- [Typescript + rollup](examples/projects/typescript-vanilla-with-rollup/README.md) - integration in a Vanilla Typescript rollup project
```
Comment (tbouffard) [inline README.md:94]: :warning: not related to this PR, probably more in #109

File: README.md [hunk 5]
```diff
@@ -73,24 +73,25 @@ the viewport horizontally or vertically
...
+- [Custom colors](examples/custom-colors/README.md) - custom defaults, specific to BPMN element types
+- [Custom fonts](examples/custom-fonts/README.md) - custom defaults, specific to BPMN element types
 - [Hacktoberfest themes](examples/hacktoberfest-diagram/README.md) - special Hacktoberfest diagram with Hacktoberfest colors
 
 
 #### interaction examples
 
 - [Open a call activity from a main BPMN Diagram on a modal](examples/interaction/call-activity-with-modal-on-mouse-over/README.md) - interact with a diagram to open the details of a Call Activity on a modal
 - [Open a call activity from a main BPMN Diagram on a tab (of the page)](examples/interaction/call-activity-with-tabs-on-click/README.md) - interact with a diagram to open the details of a Call Activity in a tab (of the page)
+- [Open a call activity from a main BPMN Diagram on the same container](./examples/interaction/call-activity-with-reload-on-dblclick/README.md) - interacted with a diagram to load the details of a Call Activity on the same container
```
Comment (tbouffard) [inline README.md:86]:
```suggestion
- [Open a call activity from a main BPMN Diagram on the same container](./examples/interaction/call-activity-with-reload-on-dblclick/README.md) - interact with a diagram to load the details of a Call Activity in the same container
```

File: examples/interaction/call-activity-with-modal-on-mouse-over/README.md [hunk 6]
```diff
@@ -10,7 +10,7 @@ After the main `BpmnVisualization` instantiation, get the HTML element correspon
 
 ⚠️  In order to avoid having to many content in the README, we simplify it. You can find all the content of the example in [index.js](index.js).
 ```javascript
-const callActivityElt = mainBpmnVisualization.htmlElementRegistry.getBpmnHtmlElement('call_activity');
+const callActivityElt = bpmnVisualization.htmlElementRegistry.getBpmnHtmlElement('call_activity');
```
Comment (tbouffard) [inline examples/interaction/call-activity-with-modal-on-mouse-over/README.md:13]: :question: why changing this as part of this PR?

File: examples/interaction/call-activity-with-reload-on-dblclick/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# Load BPMN Diagram on the same container on event on a call activity
```
Comment (tbouffard) [inline examples/interaction/call-activity-with-reload-on-dblclick/README.md:1]:
```suggestion
# Load BPMN Diagram in the same container on dobble click on a call activity
```

File: examples/interaction/call-activity-with-reload-on-dblclick/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+Javascript example to demonstrate how to use the same container on event on an element of the BPMN Diagram.
```
Comment (tbouffard) [inline examples/interaction/call-activity-with-reload-on-dblclick/README.md:3]:
```suggestion
Javascript example to demonstrate how to use the same container on a double click on an element of the BPMN Diagram.
```

File: examples/interaction/call-activity-with-reload-on-dblclick/README.md [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+- [__:fast_forward: live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/interaction/call-activity-with-reload-on-dblclick/index.html)
+- to run locally, open the [index.html](index.html) directly in a Web Browser
+
+## ♻️ Usage
+
+After the main `BpmnVisualization` instantiation, get the HTML element corresponding to the call activity to add a listener to load a new BPMN Diagram on the same container on activation.
```
Comment (tbouffard) [inline examples/interaction/call-activity-with-reload-on-dblclick/README.md:9]:
```suggestion
After the main `BpmnVisualization` instantiation, get the HTML element corresponding to the call activity to add a listener to load a new BPMN Diagram in the same container on activation.
```

**Review by tbouffard — APPROVED** [review]
LGTM, some minor comments.
I really like the breadcrumb way to switch back to the main diagram :+1:

File: examples/interaction/call-activity-with-modal-on-mouse-over/README.md (hunk 6 above)
Comment (csouchet) [reply examples/interaction/call-activity-with-modal-on-mouse-over/README.md:13]: Oops, I change the wrong file 🙁

Comment (csouchet) [reply examples/interaction/call-activity-with-modal-on-mouse-over/README.md]: Fixed

**Pull request merged**